src/main/java/de/tum/in/www1/artemis/web/rest/AccountResource.java

```diff
@@ -86,7 +86,7 @@ private boolean isSAML2Disabled() {
      * {@code POST  /register} : register the user.
      *
      * @param managedUserVM the managed user View Model.
-     * @throws InvalidPasswordException  {@code 400 (Bad Request)} if the password is incorrect.
+     * @throws PasswordViolatesRequirementsException  {@code 400 (Bad Request)} if the password is incorrect.
```

Comment:
(same below)
```suggestion
     * @throws PasswordViolatesRequirementsException  {@code 400 (Bad Request)} if the password does not meet the requirements.
```

Reply:
I would argue that the text is redundant. What do you think?

Reply:
I'd say that we should change the old text in any case because it still contains the old ambiguity. The coding guidelines do not mention JavaDoc. Afaik you should still describe every parameter and thrown exceptions shorty, even if the name is self-explanatory. The code also contains lines like `@throws EmailAlreadyUsedException {@code 400 (Bad Request)} if the email is already used.` which is the same case, so just for stake of consistency imo this short description should stay.